mapdata/WarcraftLegacies/UnitData/Skin/859189352.json

```diff
@@ -61,7 +61,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Font of arcane power. Provides 8 food, which increases the maximum number of units that can be trained.|n|cfff5962dAbilities:|R Charge Mana|n|n|cff99b4d1Can only train 48"
+      "Value": "Font of Arcane energy.|nProvides 10 food, which increases the maximum number of units that can be trained.|n|r|n|cff99b4d1Can only build 48."
```

Comment:
Dalaran's farms have no build limit, so the "Can only build 48" bit is incorrect.